### ISSUE TITLE
MineConfig.cfg file crashing game on startup

### ISSUE THREAD

**Issue by leafy-luca** [Wolfgange3311999/Mine-DS#82]
TLDR; "MineConfig.cfg" file causes startup Guru Meditation error after saving settings. Deleting the file solves the problem but resets settings. How to fix? :>

Hey there! I'm getting a "Guru Meditation" error (or something along those lines) after my first startup playing on the recommended FAT version with an Acekard i 2. 

When i first booted the game, it would load the top screen but the menu would just frantically flash stripes of red and black. I came here to search for answers and found someone commenting on the need of the files on the "data" folder at the root of the SD card (mine wasn't on the root). To my surprise the game had created it's own "data" folder, so i just moved the audio files to it, and that fixed the problem.

While playing i changed a few settings out of curiosity (such as "herobrine" and the texture pack, cause i prefer the original minecraft one), and when starting the game again, i got the following error:

![image](https://github.com/user-attachments/assets/22019f39-c268-40bb-94e2-2813e7d67428)
![image](https://github.com/user-attachments/assets/7b4acf63-3499-45b3-b36e-b512f1e5337e)

Top screen white, lower screen with the Guru error.

I deleted the "MineConfig.cfg" file on "data/Mine-DS/", because that was the only new file there so it could've been the problem, and that fixed it, the game starts now, but i have to do that for every startup and every time i have to redo my settings. **Is there any fix to this?**

**Comment by MatthewScholefield** [comment]
Hmm, interesting. Thanks for the detailed analysis! There are two paths forwards:
1. Blindly make a build of the game that doesn't load MuneConfig.cfg and see if it works for your case.
2. Put the guru meditation error into gdb and debug what is the true cause of the issue and try to fix it.

I'm kind of too lazy to do both of these 😂. But if you're interested in compiling it yourself, I can help you do option 1 on your own.
